www/src/components/chat/chat-empty-state.tsx

```diff
@@ -9,32 +10,39 @@ export function ChatEmptyState({
 }: {
   onSuggestionClick: (suggestion: string) => void;
 }) {
+  const { isMobile } = useMobileStore();
+
   return (
     <div className="flex flex-col items-center justify-center h-full p-4">
       <div className="mb-8">
-      <Image
-            src="/op-logo.svg"
-            alt="Optimism logo"
-            width={100}
-            height={100}
-          />
+        <Image
+          src="/op-logo.svg"
+          alt="Optimism logo"
+          width={75}
+          height={75}
+          className="w-[75px] md:w-[100px]"
+        />
       </div>
-      <h2 className="text-xl md:text-2xl font-bold mb-4 text-center text-chat-secondary">
-        How can I help you today?
+      <h2 className="text-lg md:text-2xl font-bold mb-4 text-center text-chat-secondary">
+        {isMobile
+          ? "Ask me anything about Optimism Governance"
+          : "How can I help you today?"}
       </h2>
-      <div className="grid grid-cols-1 md:grid-cols-2 gap-4 w-full max-w-2xl">
-        {suggestions.map((suggestion) => (
-          <Button
-            key={suggestion.label}
-            variant="outline"
-            className="flex items-center justify-start gap-2 h-auto py-3 px-4 text-sm md:text-base"
-            onClick={() => onSuggestionClick(suggestion.value)}
-          >
-            {suggestion.icon}
-            <span>{suggestion.label}</span>
-          </Button>
-        ))}
-      </div>
+      {!isMobile && (
+        <div className="grid grid-cols-1 md:grid-cols-2 gap-4 w-full max-w-2xl">
+          {suggestions.map((suggestion) => (
+            <Button
+              key={suggestion.label}
+              variant="outline"
+              className="flex items-center justify-start gap-2 h-auto py-3 px-4 text-sm md:text-base"
+              onClick={() => onSuggestionClick(suggestion.value)}
+            >
+              {suggestion.icon}
+              <span>{suggestion.label}</span>
+            </Button>
+          ))}
+        </div>
+      )}
```

Comment:
🤔 why hide all suggestions on mobile?

Reply:
<img width="347" alt="image" src="https://github.com/user-attachments/assets/97e16116-f989-461c-87c9-8c93b5fc3cbc">
I thought that it would be to much information at mobile, but it seems great tbh